Microgrant impact blog (#490)
unitaryfund/unitary.fund: src/content/blog/2024_Microgrant_Impact.md

--- 
title: "Celebrating a Milestone: Over 100 Microgrants Awarded by Unitary Fund"
author: Kallie Ferguson
day: 11
month: 6
year: 2024
--- 

> The Unitary Fund's microgrant program recently marked an incredible milestone: over 100 microgrants awarded! This achievement highlights the program's pivotal role in advancing open source quantum computing projects globally. Through strategic grants and open-source ecosystem development, Unitary Fund fosters a flourishing, multiplying effect within the broader quantum ecosystem. These small but impactful investments drive accessible and transparent progress, essential for unlocking the full potential of quantum computing.

Reviewer: ```suggestion
The Unitary Fund's microgrant program recently marked an incredible milestone: over 100 microgrants awarded! This achievement highlights the program's pivotal role in advancing open source quantum technology projects globally. Through grants and open-source ecosystem development, Unitary Fund fosters a flourishing, multiplying effect within the broader quantum ecosystem. These small but impactful investments drive accessible and transparent progress, essential for unlocking the full potential of quantum technology.
```

Thank you to our sponsors for making these microgrants possible! Core Members: [IBM Quantum](https://www.ibm.com/quantum), [DoraHacks](https://dorahacks.io/), [OQD](https://openquantumdesign.org/), and [Scientifica](https://www.scientifica.vc/); Supporting Members: [AWS](https://aws.amazon.com/braket/), [Microsoft](https://www.microsoft.com/), [Pasqal](https://www.pasqal.com/), [QC Ware](https://www.qcware.com/), [IonQ](https://ionq.com/) and [SandboxAQ](https://www.sandboxaq.com/). 

To commemorate this event, we spoke with our current and former grant recipients to gauge the impact and reach of their supported projects. Here’s what we found.

![Impact by the Numbers: 31 Countries Supported, 420+ Papers citing microgrant funded research, 40+ New Libraries, 400+ Contributors & Maintainers, 18.2k+ Commits (excluding QuTiP), 3 startups and 1 non-profit, $356k awarded in microgrants](/images/2024_microgrant_impact.png) 

> This impressive impact was obtained for less than the cost for a U.S. university to provide two fully funded PhD studentships from enrollment to graduation.  

Reviewer: @KallieFerguson Can we add this to the summary graphic? E.g. maybe at the bottom below the amount say "< the cost of funding two US PhD students"

Author: @francespoblete Would you be able to make this update?

## Impact to the Community: Growing the Ecosystem

> The survey highlighted significant contributions to the broader quantum computing community. Many projects have inspired collaborations and integrations with other tools and platforms. Shinichi Sunami’s graphix project, for example, has integrated with Perceval, fostering interoperability and expanding its utility. Xiuzhe (Roger) Luo’s grant project Yao.jl was the start of QuEra’s software development kit Bloqade. The pyZX project by Aleks Kissinger and John van de Wetering led to additional projects [QuiZX](https://github.com/quantomatic/quizx), a speedy rust port of pyzx, and [Quantomatic zxlive](https://github.com/quantomatic/zxlive), a GUI frontend. 

Reviewer: ```suggestion
The survey highlighted significant contributions to the broader quantum technology community. Many projects have inspired collaborations and integrations with other tools and platforms. Shinichi Sunami’s graphix project, for example, has integrated with Perceval, fostering interoperability and expanding its utility. Xiuzhe (Roger) Luo’s grant project Yao.jl was the start of QuEra’s software development kit Bloqade. The pyZX project by Aleks Kissinger and John van de Wetering led to additional projects [QuiZX](https://github.com/quantomatic/quizx), a speedy rust port of pyzx, and [Quantomatic zxlive](https://github.com/quantomatic/zxlive), a GUI frontend. 
```

> With a minimum of 400+ contributors across projects, Unitary Fund microgrants are useful spaces to grow our open source tooling. They represent a vibrant community of individuals contributing code to open source tools and citing microgrant work in papers over the last 6 years, creating more than 40 different libraries. 

Reviewer: ```suggestion
With over 400+ contributors across projects, Unitary Fund microgrants are useful spaces to grow our open source tooling. They represent a vibrant community of individuals contributing code to open source tools and citing microgrant work in papers over the last 6 years, creating more than 40 different libraries. 
```
